src/components/media_manager/CMakeLists.txt

```diff
@@ -32,17 +32,20 @@ include(${CMAKE_SOURCE_DIR}/tools/cmake/helpers/platform.cmake)
 include(${CMAKE_SOURCE_DIR}/tools/cmake/helpers/sources.cmake)
 
 include_directories(
-    include
+    ${COMPONENTS_DIR}/media_manager/include/
```

Comment:
@dev-gh full path of include is more obvious that related path.
But I will investigate why media manager have dependency from RC